docs/api/taxons.rst

```diff
@@ -0,0 +1,760 @@
+                        "self": {
+                            "href": "/api/v1/taxons/7"
+                        }
+                    }
+                },
+                {
+                    "id": 8,
+                    "code": "toys",
+                    "root": {
+                        "_links": {
+                            "self": {
+                                "href": "/api/v1/taxons/1"
+                            }
+                        }
+                    },
+                    "parent": {
+                        "_links": {
+                            "self": {
+                                "href": "/api/v1/taxons/1"
+                            }
+                        }
+                    },
+                    "position": 1,
+                    "translations": {
+                        "en_US": {
+                            "locale": "en_US",
+                            "id": 8,
+                            "name": "Toys",
+                            "slug": "toys"
+                        }
+                    },
+                    "images": [
+                        {
+                            "id": 1,
+                            "type": "ford",
+                            "path": "b9/65/01cec3d87aa2b819e195331843f6.jpeg"
+                        }
+                    ],
+                    "_links": {
+                        "self": {
+                            "href": "/api/v1/taxons/8"
+                        }
+                    }
+                }
+            ]
+        }
+    }
+
+.. note::
+
+    The images should be passed in array as an attribute (files) of request. See how it is done in Sylius
+    `here <https://github.com/Sylius/Sylius/blob/master/tests/Controller/TaxonApiTest.php>`_.
+
+Updating Taxon
+--------------
+
+To fully update a taxon you will need to call the ``/api/v1/taxons/taxon_id`` endpoint with ``PUT`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    PUT /api/v1/taxons/{id}
+
++------------------------------------+----------------+--------------------------------------+
+| Parameter                          | Parameter type | Description                          |
++====================================+================+======================================+
+| Authorization                      | header         | Token received during authentication |
++------------------------------------+----------------+--------------------------------------+
+| id                                 | url attribute  | Id of the requested taxon            |
++------------------------------------+----------------+--------------------------------------+
+|translations['locale_code']['name'] | request        | Name of the taxon                    |
++------------------------------------+----------------+--------------------------------------+
+|translations['locale_code']['slug'] | request        | **(unique)** Slug                    |
```

Comment:
You mean that here should be also `translations['locale_code']['description'] ` ?